### PR TITLE
[16.0][ADD] sell_only_by_packaging

### PR DESCRIPTION
**Pull request by QuocDuong1306**
This module supports 2 features based on `sale_by_packaging  v14`:
- Sell only by packaging: On product template model, this checkbox restricts
  sales of these products if no packaging is selected on the sale order line.
  If no packaging is selected, it will either be auto-assigned if the quantity
  on the sale order line matches a packaging quantity or an error will be raised.

- Force sale quantity (on the packaging): force rounds up the quantity during
  creation/modification of the sale order line with the factor set on the packaging.

### CODE REVIEW

**Comment by nilshamerlinck** [comment]
do you face this error in tests?

```
2023-07-27 12:05:23,142 886998 ERROR v16c_sell_only_by_packaging odoo.addons.sell_only_by_packaging.tests.test_sale_line_onchanges: FAIL: TestPackaging.test_compute_qties
Traceback (most recent call last):
  File "/home/nils/code/oca/sale-workflow/16.0-add-sell_only_by_packaging/sell_only_by_packaging/tests/test_sale_line_onchanges.py", line 13, in test_compute_qties
    line.product_packaging_id = self.packaging_tu
  File "/home/nils/code/native/repos/odoo/16.0/odoo/tests/common.py", line 2222, in __setattr__
    assert descr is not None, "%s was not found in the view" % field
AssertionError: product_packaging_id was not found in the view
```

**Comment by QuocDuong1306** [comment]
hi @nilshamerlinck , I updated

File: sell_only_by_packaging/readme/CREDISTS.rst
Comment (nilshamerlinck) [inline sell_only_by_packaging/readme/CREDISTS.rst:1]: CREDITS.rst

File: sell_only_by_packaging/models/sale_order_line.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                    line.product_packaging_qty,
+                    int(line.product_packaging_qty),
+                    precision_digits=2,
+                )
+                != 0
+            ):
+                raise ValidationError(
+                    _(
+                        "Product %s can only be sold with a packaging and a "
+                        "packaging qantity."
```
Comment (nilshamerlinck) [inline sell_only_by_packaging/models/sale_order_line.py]:
```suggestion
                        "packaging quantity."
```

**Review by nilshamerlinck — CHANGES_REQUESTED** [review]
nitpick: can you make sure that all *.py file have an empty line after the copyright?

**Pull request closed without merging**